tests/topotests/zebra_pref64/rx_ipv6_ra_8781.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python
```

Comment:
Still fails as  `rx_ipv6_ra_8781.py`  has `#!/usr/bin/python` instead of `#!/usr/bin/env python` line at the beginning